extensions/amp-analytics/0.1/vendors.js

```diff
@@ -454,6 +454,59 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'mparticle': {
+    'vars': {
+      'apiKey': '',
+      'eventName': '',
+      'eventType': 'Unknown',
+      'eventAttributes_Keys': [],
```

Comment:
Good point. But I just kept those so that developers will know all the variables that they can set.